File: Codeforce/R208-2/B.py
n = input()
s = "<3"
for i in xrange(n):
    s += raw_input()
    s += "<3"
ans = raw_input()
slen = len(s)
alen = len(ans)
index = 0
for i in xrange(alen):
    if ans[i] != s[index]:
        continue
    index += 1
    if index == slen:
        break
if index == slen:
    print "yes"
else:
    print "no"

File: Codeforce/R208-2/B.rb
n = gets.to_i
pattern = "<3"
n.times do |i|
  pattern = pattern << gets.chomp
  pattern = pattern << "<3"
end
ans = gets
len = pattern.size()
index = 0
alen = ans.size()
alen.times do |i|
  next if (pattern[index]!=ans[i])
  break if index == len or alen - i < len - index
  index += 1
end
if index == len then
  puts "yes"
else
  puts "no"
end
